ambry-commons/src/main/java/com.github.ambry.commons/HelixNotifier.java

```diff
@@ -142,6 +144,32 @@ public void unsubscribe(String topic, TopicListener<String> topicListener) {
   }
 
   /**
+   * A helper method that gets a {@link HelixNotifier}. Each call of this method will generate a different
+   * {@link HelixNotifier} instance.
+   * @param storeConfig A {@link HelixPropertyStore} used to instantiate a {@link HelixNotifier}.
+   * @return A {@link HelixNotifier}.
+   */
+  public static HelixNotifier getHelixNotifier(HelixPropertyStoreConfig storeConfig) {
```

Comment:
This method serves as a constructor for `HelixNotifier`. The reason not to put it into a constructor depends on if we want to control the helix property store the notifier depends on. I see values of doing on either side, but have some decoupling is not a bad thing, as we can control what to pass in. However, the caller may not want to get involved into the complication of instantiating a propertystore, so we introduce this helper method.